packages/arb-token-bridge-ui/src/hooks/useArbQueryParams.tsx

```diff
@@ -40,7 +41,8 @@ export const useArbQueryParams = () => {
     l2ChainId: NumberParam, // L2 chain-id with which we can initiaze (override) our networks/signer
     token: StringParam, // import a new token using a Dialog Box
     // todo: use custom union type param for theme for better types
-    theme: withDefault(StringParam, '') // for switching to 'arbitrum-classic-theme'
+    theme: withDefault(StringParam, ''), // for switching to 'arbitrum-classic-theme'
+    stats: withDefault(BooleanParam, false) // for switching on 'arbitrum-stats'
```

Comment:
so i know the default is `?qp=1` for `BooleanParam` but I prefer it to be `?qp=true`
just a comment though, we should probably just keep it as it is